style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```

Comment:
- [OPTIONAL]  The same rule goes for this as regards the comment I made for your `h1` element. It is a best practice to use class selector rather than element selector. It is for semantics reason. Kindly use the class name given to your `h1` element to style its content.